content/en/mpas/getting_started.md

```diff
@@ -0,0 +1,394 @@
+kind: Target
+metadata:
+  name: podinfo-kubernetes-target
+  namespace: mpas-podinfo-application
+  labels:
+    target.mpas.ocm.software/ingress-enabled: "true" # DO NOT CHANGE. expected by the component for target selection.
+spec:
+  type: kubernetes
+  access:
+    targetNamespace: podinfo
+EOF
+```
+
+Then, apply the `Target` to the project in a gitOps fashion:
+
+```bash
+git add --all && git commit -m "Add a target for podinfo" && git push
+```
+
+`Flux` will detect the changes and apply the target to the cluster.
+
+7. Deploy the podinfo application
+
+In order to deploy the podinfo application, we need to create a `ProductDeploymentGenerator` resource:
+
+```bash
+cat <<EOF >> ./generators/podinfo.yaml
+apiVersion: mpas.ocm.software/v1alpha1
+kind: ProductDeploymentGenerator
+metadata:
+  name: podinfo
+  namespace: mpas-podinfo-application
+spec:
+  interval: 1m
+  serviceAccountName: mpas-podinfo-application
+  subscriptionRef:
+    name: podinfo-subscription
+    namespace: mpas-podinfo-application
+EOF
+```
+
+Then, apply the `ProductDeploymentGenerator` to the project in a gitOps fashion:
+
+```bash
+git add --all && git commit -m "Add podinfo deployment generator" && git push
+```
+
+`Flux` will detect the changes and apply the resource to the cluster.
+
+This will create a pull request in the project repository with the `ProductDeployment` resource
+that will deploy the podinfo application.
+
+<!--TODO: The pull request commit needs changed to something more meaningful--->
+
+Go to the project repository and retrieve the pull request. 
+It should contains a `ProductDeployment` declaration that provides the configuration and
+all steps needed to deploy the product, as well as a `values.yaml` file. The `values` file
+contains values that should used to configure the different resources that are part of
+the product to be deployed. There is a check that should pass before merging the pull request.
+
+Once the pull request is merged, `Flux` will detect the changes and deploy the application to the cluster.
+
+After a moment the `ProductDeployment` should be deployed successfully.
+It is possible to verify with the command:
+
+```bash
+k describe productDeployment -n mpas-podinfo-application  
+```
+
+The result should something like:
+
+```bash
+Name:         podinfo
+Namespace:    mpas-podinfo-application
+Labels:       kustomize.toolkit.fluxcd.io/name=mpas-podinfo-application-products
+              kustomize.toolkit.fluxcd.io/namespace=mpas-system
+API Version:  mpas.ocm.software/v1alpha1
+Kind:         ProductDeployment
+Metadata:
+...
+Status:
+  Conditions:
+    Last Transition Time:  2023-09-14T10:14:41Z
+    Message:               Reconciliation success
+    Observed Generation:   1
+    Reason:                Succeeded
+    Status:                True
+    Type:                  Ready
+  Observed Generation:     1
+```
+
+You should then have the application deployed in the `podinfo` namespace.
```

Comment:
Hm, these shoulds read a bit differently. Maybe be a bit more assertive?

Is deployed, contains, etc...? That would read more decisive. WDYT?